bedrock/firefox/templates/firefox/whatsnew/whatsnew-2019.html

```diff
@@ -0,0 +1,79 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_2019" %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{% block page_title %}{{ _('With Firefox, your privacy comes first') }}{% endblock %}
+{% block page_desc %}{{ _('Everything we make and do honors our Personal Data Promise: Take less. Keep it safe. No secrets.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_2019') }}
+{% endblock %}
+
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="mzp-t-firefox content-wrapper {% block container_class %}{% endblock %}">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+    {% block header_logo %}
+      {{ high_res_img('logos/firefox/logo-quantum-wordmark-large.png', {'alt': 'Firefox', 'width': '122', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+    {% endblock %}
+      <h1 class="c-page-header-up-to-date"><span>{{ _('Congrats! You’re using the latest version of Firefox.') }}</span></h1>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+    {% block content_main %}
+      <div class="c-emphasis-box show-fxa-default">
+        <h2 class="c-emphasis-box-title">{{ _('Get the respect you deserve') }}</h2>
+        <p>
+        {% trans promise='#' %}
```

Comment:
Oh right, this proved to be a little tricky since there are three URLs for three blog posts. For the accounts page we did it in the view and we should do the same here, but this view is much more complicated and I had trouble getting the context right so I punted and forgot to go back to it. I'll work on this.

Reply:
Removed this template